beacon-chain/rpc/validator/proposer.go

```diff
@@ -29,6 +29,8 @@ import (
 
 // eth1DataNotification is a latch to stop flooding logs with the same warning.
 var eth1DataNotification bool
+var cachedEth1VotingStartTime uint64
+var cachedEth1Data *ethpb.Eth1Data
```

Comment:
This caching is very simple, should we cache it on the beacon node side or is on the validator end fine?

Reply:
It should be fine if it lives in the beacon node side, that way many validator clients connected to one beacon node can benefit